metrics/resizer.go

```diff
@@ -25,6 +25,11 @@ func (a *resizerSuccessResizeTotalAdapter) Increment(pvcname string, pvcns strin
 	a.metric.With(prometheus.Labels{"persistentvolumeclaim": pvcname, "namespace": pvcns}).Inc()
 }
 
+// This method is for outputting metrics before the first resize.
+func (a *resizerSuccessResizeTotalAdapter) Nop(pvcname string, pvcns string) {
```

Comment:
I feel strange that this method is named as Nop. It may create new metric or label, i.e., it has effect.

Reply:
I updated it.